packages/eslint-plugin/tests/docs.test.ts

```diff
@@ -252,9 +252,17 @@ describe('Validating rule docs', () => {
         // Get all H2 headings objects as the other levels are variable by design.
         const headings = tokens.filter(tokenIsH2);
 
-        headings.forEach(heading =>
-          expect(heading.text).toBe(titleCase(heading.text)),
-        );
+        headings.forEach(heading => {
+          // Hard-code exceptions for code in headings.
+          if (
+            heading.text ===
+            'Explicit Resource Management (`await using` Statements)'
+          ) {
+            return;
+          }
+
+          expect(heading.text).toBe(titleCase(heading.text));
+        });
```

Comment:
Ehh, this is super specific to the heading. I think we'd be better off generalizing. How about removing anything inside code ticks with a regex?

```suggestion
        headings.forEach(heading => {
          const nonCodeText = heading.text.replace(/`[^`]*`/g, '');
          expect(nonCodeText).toBe(titleCase(nonCodeText));
        });
```

Reply:
oh, yeah, much better 👍